images/macos/tests/Common.Tests.ps1

```diff
@@ -304,33 +320,23 @@ Describe "Haskell" -Skip:($os.IsHighSierra) {
     }
 }
 
-Describe "Clang/LLVM" -Skip:($os.IsHighSierra) {
-    It "Clang/LLVM is installed" {
-        "$(brew --prefix llvm)/bin/clang --version" | Should -ReturnZeroExitCode
-    }
-}
-
-Describe "Gcc" -Skip:($os.IsHighSierra) {
-    $testCases = @("8", "9", "10") | ForEach-Object { @{GccVersion = $_} }
+Describe "gcc" -Skip:($os.IsHighSierra) {
+    $testCases = @("8", "9", "10") | ForEach-Object { @{Version = $_} }
 
     It "Gcc <GccVersion>" -TestCases $testCases {

```

Comment:
I think `<GccVersion>` you need to change it to `<Version>`. Same for other test cases as well.

Reply:
Thank you Dibir, good point. I've fixed this.